src/main/scala/lms/transformation/mpi_nccl_transformation/distributedTensor2MPINCCL.scala

```diff
@@ -33,6 +33,9 @@ class DistributeTensor2MPI_NCCLAnalysis extends Traverser {
         case Node(s, op, _, _) if op.startsWith("tensor_conv") =>
             hasCudnn = true
             super.traverse(n)
+        case Node(s, op, _, _) if op.startsWith("tensors_dropout") =>
+            hasCudnn = true
+            super.traverse(n)
```

Comment:
maybe we should have a list of `cudnn_ops` and just do `if cudnn_ops.contains(op)`

Reply:
I was thinking the same thing. Maybe we can fix that after softmax and dropout are merged, together with other fixes.